physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -210,17 +210,17 @@
   kind = kind_phys
   intent = in
 [nsoil]
-  standard_name = vertical_dimension_of_soil
-  long_name = soil vertical layer dimension
+  standard_name = vertical_dimension_of_soil_internal_to_land_surface_scheme
+  long_name = number of soil layers internal to land surface model
   units = count
   dimensions = ()
   type = integer
   intent = in
 [smc]
-  standard_name = volume_fraction_of_condensed_water_in_soil
-  long_name = volumetric fraction of soil moisture
+  standard_name = volume_fraction_of_soil_moisture_for_land_surface_model
+  long_name = volumetric fraction of soil moisture for lsm
   units = frac
-  dimensions = (horizontal_loop_extent,vertical_dimension_of_soil)
+  dimensions = (horizontal_dimension,vertical_dimension_of_soil_internal_to_land_surface_scheme)
```

Comment:
@haiqinli This might be the cause of the error that you're getting. The horizontal dimension for variables within the `run` phase of CCPP should be `horizontal_loop_extent`.

Reply:
@grantfirl Thank you very much for your help. I will rerun the regression test with this fix.